src/System.Text.Primitives/System/Text/Encoding/Utf16/Utf16TextEncoderLE.cs

```diff
@@ -152,6 +152,29 @@ public override bool TryEncode(string text, Span<byte> encodedBytes, out int byt
             return true;
         }
 
+        public override bool TryComputeEncodedBytes(ReadOnlySpan<byte> utf8, out int bytesNeeded)
+        {
+            int characters;
+            var result = Utf8Encoder.TryComputeStringLength(utf8, out characters);
```

Comment:
addressed in https://github.com/dotnet/corefxlab/pull/1467/commits/a9a906bfedb0b626ae4f355966326aa634b96a6e